cypress/support/commands.ts

```diff
@@ -24,6 +25,8 @@ const ethAccount = makeEthAccount()
 const btcAccount = makeBtcAccount()
 const ethUsdcSwapRate = makeEthUsdcRateResponse()
 const ethFoxSwapRate = makeEthFoxRateResponse()
+const foxEthSwapRate = makeFoxEthSwapRateResponse()
+const usdcFoxSwapRate = makeUsdcFoxSwapRateResponse()
```

Comment:
I agree with this suggestion 👌 it would add an extra element of purity to the tests - I'll add that in when I'm back at the keyboard tomorrow, thanks @gomesalexandre!

Reply:
Refactored accordingly in https://github.com/shapeshift/web/pull/1012/commits/467ab44052dab74ed02953e0645fddd1279140bf